packages/taco/src/conditions/context/context.ts

```diff
@@ -119,12 +129,20 @@ export class ConditionContext {
     // First, we want to find all the parameters we need to add
     const requestedParameters = new Set<string>();
 
-    // Search conditions for parameters
     // Check return value test
     if (condition.returnValueTest) {
       const rvt = condition.returnValueTest.value;
-      if (ConditionContext.isContextParameter(rvt)) {
+      // Return value test can be a single parameter or an array of parameters
+      if (Array.isArray(rvt)) {
+        rvt.forEach((value) => {
+          if (ConditionContext.isContextParameter(value)) {
+            requestedParameters.add(value);
+          }
+        });
+      } else if (ConditionContext.isContextParameter(rvt)) {
         requestedParameters.add(rvt);
+      } else {
+        // Not a context parameter, we can skip
       }
```

Comment:
Is the empty block for a comment something common in typescript? Perhaps just the comment?
```suggestion
      }
      // else not a context parameter, we can skip
```

Reply:
It's no TS-related, just something I sometimes do to be more explicit.